regnety/dataset/imagenet.py

```diff
@@ -227,10 +239,17 @@ def _scale_to_unit(self, image: tf.Tensor, target: tf.Tensor) -> tuple:
         Returns:
             Scaled example with batch of images and targets with same dimensions.
         """
-
-        aug_images = tf.cast(image, tf.float32)
-        aug_images = aug_images / 255.
-        return aug_images, target
+        if self.scale_method == "torch":
+            aug_images = tf.cast(image, tf.float32)
+            aug_images = aug_images / 127.5
+            aug_images = aug_images - 1.
+            return aug_images, target
+        
+        else:
+            aug_images = tf.cast(image, tf.float32)
+            aug_images = aug_images / 255.
+            return aug_images, target
+            
```

Comment:
Aren't we doing this inside the model? And if so, I think it's better to remove it.

Reply:
No, 'cause there's no Rescaling layer in TF Keras 2.5. It is present only in the nightly version, I think. THus, it needs to be done in preprocessing pipeline itself.

Reply:
I am confused by this one: https://www.tensorflow.org/api_docs/python/tf/keras/layers/experimental/preprocessing/Rescaling. It's 2.5.0. In fact, it was there since 2.4.1. Or am I missing something?

Reply:
Sorry, I was misled by [this](https://user-images.githubusercontent.com/64411306/126889024-f1bc2582-f8c2-489b-ad2d-3e8c732d7f38.png). I'll change that asap.

Reply:
In this case, do we want to keep just one of them, the /255. one? Or should I include both?

Reply:
If we are mean-subtracting the channels then /255 is anyway required to make the value range suitable. Even if the mean-subtraction part is omitted just scaling pixel values to [0, 1] is still a widely used practice I'd say.